app/test_main.py

```diff
@@ -1,3 +1,19 @@
+import pytest
 from app.main import get_human_age
 
-# write your code here
+
+def test_incorrect_year_format() -> None:
+    with pytest.raises(ValueError):
+        get_human_age(-12, 43.1)
+
+
+def test_with_zero_expected():
+    assert get_human_age(12, 14) == [0, 0]
```

Comment:
The expected result `[0, 0]` seems incorrect based on the logic in `convert_to_human`. For `cat_age` 12 and `dog_age` 14, the function should return `[1, 1]` according to the conversion logic. Please verify and update the expected results.